src/UserBundle/Tests/Repository/UserInvitationRepositoryTest.php

```
@@ -0,0 +1,91 @@
+            ->setCompany($this->company)
+            ->setStatus(UserInvitation::STATUS_PENDING);
+        $this->repository->save($invitation2);
+
+        self::assertSame(2, $this->repository->countPendingInvitations());
+    }
+
+    public function testGetGridQuery(): void
+    {
+        $queryBuilder = $this->repository->getGridQuery();
+        self::assertInstanceOf(QueryBuilder::class, $queryBuilder);
+        $alias = $queryBuilder->getRootAliases()[0];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$alias'. (undefined)

(UnusedLocalVariable)